gp-includes/assets-loader.php

```diff
@@ -36,14 +37,16 @@ function gp_register_default_scripts() {
 	wp_register_script( 'gp-glossary', $url . '/glossary' . $suffix, array( 'gp-editor' ), filemtime( $path . 'glossary' . $suffix ) );
 	wp_register_script( 'gp-translations-page', $url . '/translations-page' . $suffix, array( 'gp-editor' ), filemtime( $path . 'translations-page' . $suffix ) );
 	wp_register_script( 'gp-mass-create-sets-page', $url . '/mass-create-sets-page' . $suffix, array( 'gp-editor' ), filemtime( $path . 'mass-create-sets-page' . $suffix ) );
+	wp_register_script( 'gp-jquery-webui-popover', $url . '/jquery.webui-popover' . $suffix, array( 'jquery' ), filemtime( $path . 'jquery.webui-popover' . $suffix ), false );
 	wp_register_script( 'gp-tour', $url . '/tour' . $suffix, array( 'jquery', 'gp-jquery-webui-popover' ), filemtime( $path . 'tour' . $suffix ), false );
+	wp_register_script( 'driver-js', $url . '/driver-js' . $suffix, array(), '1.3.0' );
```

Comment:
Please, change the hard-coded version to a one based in the filemtime:

```suggestion
	wp_register_script( 'driver-js', $url . '/driver-js' . $suffix, array(), filemtime( $path . 'driver-js' . $suffix ) );

```